RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -497,8 +499,6 @@ boolean L13_towerNSContests()
 				}
 				break;
 			case $stat[mysticality]:
-				autoMaximize("myst -equip snow suit", 1500, 0, false);
-
 				if(have_effect($effect[Uncucumbered]) == 0 && auto_is_valid($effect[Uncucumbered]))
```

Comment:
This just refactors out the 3 cases to converting the stat to a string and adding it into the maximizer that way, since other than stat name the 3 autoMaximize calls don't differ at all.